PyP100/PyP100.py

```diff
@@ -0,0 +1,354 @@
+import requests
+from requests import Session
+
+from base64 import b64encode, b64decode
+import hashlib
+from Crypto.PublicKey import RSA
+import time
+import json
+from Crypto.Cipher import AES, PKCS1_OAEP, PKCS1_v1_5
+from . import tp_link_cipher
+import ast
+import pkgutil
+import uuid
+import json
+
+
+#Old Functions to get device list from tplinkcloud
+def getToken(email, password):
+	URL = "https://eu-wap.tplinkcloud.com"
+	Payload = {
```

Comment:
There are quite a few places in the code where the URL and Payload variables are used. Since they contain similar string values, perhaps we could compose these values into a separate structure and use them as a DSL? It could be a simple dictionary, or a json file, for example.